src/apis/friends/index.ts

```
@@ -44,7 +44,7 @@ export const useDeleteFriend = (option: MutationOptions<void, Error, string>) =>
   })
 }
 
-export const useGetMyFriendList = (name: string) => {
+export const useGetMyFriendList = (name: string, p0?: { enabled: boolean }) => {
```

ESLint

[error] 'p0' is defined but never used.

(@typescript-eslint/no-unused-vars)